docs/DeveloperGuide.adoc

```diff
@@ -502,8 +533,46 @@ A separate parser for phone, email, address and remark are used to accommodate t
 **Alternative 2:** update the tag to accept sentences for remarks +
 **Pros:** Easy to implement as we do not need to add a new command. +
 **Cons:** The Tag may be more difficult to find and more work is needed to change the find tag command. Additionally, the user interface of tag command also needs to be changed to accommodate long tags. +
+
+---
+
+**Aspect:** Defining a valid remark input +
+**Alternative 1 (current choice):** No restrictions on the kind of remark input the user can put into the addressbook, i.e. the user can put in any characters in any format. +
+**Pros:** More freedom for the user as the user can type in anything for adding a remark. It is also easier to implement as the input string does not have to go through other checks. +
+**Cons:** Can get messy if the user is not consistent in the way they place in remarks. The sort command applied under remarks also does not sort the remark as well if the input is not consistent. +
+**Alternative 2:** Impose a restriction on remarks in the format [Remark Type] remark followed by restriction of only allowing alphabets. +
+**Pros:** . The sort remark command will be able to organise and sort the remark better since each remarks are predefined at the start in the [Remark Type].+
+**Cons:** . More difficult to implement as we need additional checks to be placed for the restrictions and also less freedom for the user as they need to follow stricter rules before putting in a remark entry.+
```

Comment:
don't need plus sign at the end of cons

Reply:
The other fields have a plus sign at the end of the cons. Eg UndoRedo